README.md

```diff
@@ -0,0 +1,70 @@
+# Dev environemnt setup
+
+This file is an explanation on how to setup the development environment for PC subteam projects.
+
+I recommend that you write all the commands by hand instead of copying them directly from this file.
+
+## Prerequisites:
+- Basic use of a terminal with: cd, ls, mkdir, pwd
```

Comment:
Use `whatis` or `man -f` to see one-line explanations of the following commands:
- `man`
- `whatis`
- `pwd`
- `cd`
- `ls`
- `echo`
- `mkdir`
- `rmdir`
- `touch`

Familiarize yourself with using a shell:
```bash
cd ~
pwd
cd ..
pwd
whatis date
man date
echo "Now is $(date) - the best time since $(($(date +%Y) - 333) to start using the terminal :-)"
```

Reply:
> `echo "Now is $(date) - the best time since $(($(date +%Y) - 333) to start using the terminal :-)"`
I dont realy understand the syntax here and there is a syntax error